exasol_transformers_extension/resources/templates/sequence_classification_udf.jinja.sql

```diff
@@ -0,0 +1,5 @@
+CREATE OR REPLACE {{ language_alias }} SCALAR SCRIPT "TE_SEQUENCE_CLASSIFICATION_UDF"(...)EMITS  (outputs VARCHAR(2000000)) AS
```

Comment:
Don't use a scalar UDF, because it is difficult to control the number of instances for them. You need also add the iterating over the rows to the UDF.